apps/wfo-ui/translations/README.md

```diff
@@ -0,0 +1,44 @@
+## Adding or overriding translations
```

Comment:
Since the translations are part of the library as well, we could also add/move it there